packages/manager/apps/pci-users/src/pages/EditRolesPage.tsx

```diff
@@ -12,7 +12,7 @@ export default function EditRolesPage() {
   const onClose = () => {
     navigate('..');
   };
-  const userId = searchParams.get('userId') as number | null;
+  const userId = parseInt(searchParams.get('userId'), 10) || null;
```

Comment:
i think it won't matter in this case but keep in mind that "0" will be "null" with this